### DISCUSSION TITLE
gateway not take user IP

### DISCUSSION THREAD

**Discussion by legendskyk** [inspircd/inspircd#2109] [Support]
I set webirc password and mask still user ip not getting it user join through default ip what is that issue?

**Comment by sadiepowell** [comment]
Can you pastebin your config (you can censor the password)?

**Reply by legendskyk** [reply]
<gateway type="webirc" password="kwIvi87KiCRcri876Hjklow" mask="139.99.135.109">

**Comment by legendskyk** [comment]
<gateway type="webirc" password="kwIvi87KiCRcri876Hjklow" mask="139.99.135.109">

**Reply by legendskyk** [reply]
https://pastebin.com/avpKsBCN

**Reply by sadiepowell** [reply]
1. Are you using v4? `<gateway>` is called `<cgihost>` in v3.

2. Do you have the gateway module loaded?

**Reply by legendskyk** [reply]
yes v4

**Reply by legendskyk** [reply]
yes module is loaded

**Reply by sadiepowell** [reply]
Oops I managed to lose this thread. Did you manage to get it working?

**Comment by legendskyk** [comment]
you want to see config code of kiwiirc?
